test/erp12/ga_clj/toolbox_test.cljc

```diff
@@ -0,0 +1,24 @@
+(ns erp12.ga-clj.toolbox-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [erp12.ga-clj.toolbox :refer [lexicase-selection]]))
+
+(def population [{:name :A :errors [10 5 5 15 10]}
+                 {:name :B :errors [8 7 8 8 7]}
+                 {:name :C :errors [73 60 0 0 1]}
+                 {:name :D :errors [15 12 14 15 1]}
+                 {:name :E :errors [15 12 0 106 1]}])
```

Comment:
This should probably be either 1) moved into a `let` inside the `deftest` or 2) renamed to something like `population-with-error-vectors` because this population will only be used when testing certain algorithms.